components/lib/autocomplete/AutoComplete.js

```diff
@@ -159,7 +159,7 @@ export const AutoComplete = React.memo(
         };
 
         const formatValue = (value) => {
-            if (value) {
+            if (value || value === 0) {
```

Comment:
I think you can use ObjectUtils.IsNotEmpty(value)

Reply:
Done, thanks. @melloware